Jenkinsfile

```diff
@@ -38,7 +38,9 @@ pipeline {
                     }
                     steps {
                         checkout scm
-                        sh "make build"
```

Comment:
You can turn this into a single sh invocation

```
sh '''make build
make test
docker system prune --force --all
'''
```

Reply:
Done